### ISSUE TITLE
Importing order is not working consistently across OSs (MacOs)

### ISSUE THREAD

**Issue by caiokoiti** [benmosher/eslint-plugin-import#1849]
Hello Dear,

Its similar to this issue:
https://github.com/benmosher/eslint-plugin-import/issues/1692

Running the lint here on macs it complains about the importing order.

Running on mohave.
nodejs 12.18.2
"eslint": "^7.3.1"
"eslint-plugin-import": "^2.18.2"

**Comment by ljharb** [comment]
Can you provide the code that you're linting, and the error that you're seeing?

**Comment by caiokoiti** [comment]
@ljharb,

Sure thing.

### MacOS
![Screenshot 2020-07-15 10 36 04](https://user-images.githubusercontent.com/2758613/87496948-21500b80-c687-11ea-904e-21386db5d25c.png)

![Screenshot 2020-07-15 10 40 19](https://user-images.githubusercontent.com/2758613/87497174-a63b2500-c687-11ea-988c-eda5d37bd8f9.png)


### windows

**Comment by nitpum** [comment]
same here

**Comment by ljharb** [comment]
This would be easiest to debug with either a PR with a failing test case, or with a repro repo.

**Comment by caiokoiti** [comment]
This is still an issue, right?

**Comment by ljharb** [comment]
@caiokoiti yes, i still haven't found a way to reproduce it. A failing test or repro repo would help tremendously.

**Comment by Fsss126** [comment]
Experiencing the same problem. I get import errors on Mac and get no errors on remote development server with Linux
